spec/controllers/cidadaos_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe CidadaosController do
+  let!(:user) { create(:user) }
+  before { sign_in user }
+
+  describe 'GET #index' do
+    let(:cidadaos) { create_list(:cidadao, 5) }
+
+    before { get :index }
+
+    it 'validate status' do
+      expect(response).to  have_http_status(:success)
+    end
+
+    it 'validate render template' do
+      expect(response).to  render_template(:index)
+    end
+  end
+
+  describe 'GET #new' do
+    before { get :new }
+
+    it 'validate status' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'validate render template' do
+      expect(response).to render_template(:new)
+    end
+
+    it 'validate new object' do
+      expect(assigns(:cidadao)).to be_a_new(Cidadao)
+    end
+  end
+
+  describe 'POST #create' do
+    let(:cidadao_params) { attributes_for(:cidadao) }
+
+    context 'When cidadao have valid params', aggregate_failures: true do
+      it 'validate create object' do
+        expect{
+          post :create, params: { cidadao: cidadao_params }
+            }.to change(Cidadao, :count).by(1)
+      end
+
+      it 'validate status' do
+        post :create, params: { cidadao: cidadao_params }
+
+        expect(response).to have_http_status(302)
```

Comment:
Porque testar o 302? a linha abaixo já está fazendo isso. Quando usando `expect(response).to redirect_to` é SEMPRE um 302! [Ver doc](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Status/302)